src/nvim/normal.c

```diff
@@ -4500,7 +4500,7 @@ static void nv_zet(cmdarg_T *cap)
        * off this fails and find_ident_under_cursor() is
        * used below. */
       emsg_off++;
-      len = spell_move_to(curwin, FORWARD, true, true, NULL);
+      len = (size_t)spell_move_to(curwin, FORWARD, true, true, NULL);
```

Comment:
I guess this is what fails the test but changing len to `ptrdiff_t` is hard.

Reply:
would it help to disable `-Wconversion` for now?